### ISSUE TITLE
JsTestDriver plugin for WebStorm intermittently generates exceptions when configuration is executed

### ISSUE THREAD

**Issue by GoogleCodeExporter** [manojjsm/js-test-driver#165]
```
What steps will reproduce the problem?
1. Start the server from the plugin UI
2. Capture a few browsers
3. Run the JsTestDriver configuration

What is the expected output? What do you see instead?
The expected output is the test results, but quite often I will get "No tests 
were found." message in the main test window, even when tests haven't changed, 
and the console will show this:

Testing started at 10:18 AM ...
Nov 18, 2010 10:18:43 AM com.google.jstestdriver.ActionRunner runActions
INFO: Running com.google.jstestdriver.BrowserActionsRunner@19c5466b
Nov 18, 2010 10:18:43 AM com.google.jstestdriver.ActionRunner runActions
INFO: Finished com.google.jstestdriver.BrowserActionsRunner@19c5466b
Nov 18, 2010 10:18:43 AM com.google.jstestdriver.ActionRunner runActions
INFO: Running com.google.jstestdriver.FailureCheckerAction@23e0512a
Nov 18, 2010 10:18:43 AM com.google.jstestdriver.ActionRunner runActions
INFO: Finished com.google.jstestdriver.FailureCheckerAction@23e0512a
Nov 18, 2010 10:18:43 AM com.google.jstestdriver.ActionRunner runActions
INFO: 
Nov 18, 2010 10:18:43 AM com.google.jstestdriver.ActionRunner runActions
INFO: Running com.google.jstestdriver.BrowserActionsRunner@5bf0aecc
Exception in thread "main" 
Test Errors:
    atFailures during test run.
    at com.google.jstestdriver.BrowserActionsRunner.run(BrowserActionsRunner.java:108)
    at com.google.jstestdriver.ActionRunner.runActions(ActionRunner.java:56)
    at com.google.jstestdriver.idea.TestRunner.execute(Unknown Source)
    at com.google.jstestdriver.idea.TestRunner.main(Unknown Source)
Caused by:
java.lang.RuntimeException: java.net.SocketException: Broken 
pipejava.lang.RuntimeException: java.net.SocketException: Broken pipeEmpty test 
suite.


What version of the product are you using? On what operating system?
I am using WebStorm 2.0 EAP build 98.364 on Mac OSX 10.6.5 with plugin v.1.2.2a

Please provide any additional information below.

IDE also reports a critical internal error: 

java.io.UTFDataFormatException: java.io.UTFDataFormatException
java.lang.RuntimeException: java.io.UTFDataFormatException
    at com.google.jstestdriver.idea.RemoteTestListener$1.finished(Unknown Source)
    at com.intellij.util.concurrency.SwingWorker$1.run(SwingWorker.java:148)
    at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:319)
    at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:209)
    at java.awt.EventQueue.dispatchEvent(EventQueue.java:633)
    at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:655)
    at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:509)
    at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:408)
    at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:372)
    at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:296)
    at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:211)
    at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:201)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:196)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:188)
    at java.awt.EventDispatchThread.run(EventDispatchThread.java:122)
Caused by: java.io.UTFDataFormatException
    at java.io.ObjectInputStream$BlockDataInputStream.readUTFSpan(ObjectInputStream.java:3081)
    at java.io.ObjectInputStream$BlockDataInputStream.readUTFBody(ObjectInputStream.java:3006)
    at java.io.ObjectInputStream$BlockDataInputStream.readUTF(ObjectInputStream.java:2819)
    at java.io.ObjectInputStream.readUTF(ObjectInputStream.java:1050)
    at java.io.ObjectStreamClass.readNonProxy(ObjectStreamClass.java:649)
    at java.io.ObjectInputStream.readClassDescriptor(ObjectInputStream.java:808)
    at java.io.ObjectInputStream.readNonProxyDesc(ObjectInputStream.java:1564)
    at java.io.ObjectInputStream.readClassDesc(ObjectInputStream.java:1495)
    at java.io.ObjectInputStream.readOrdinaryObject(ObjectInputStream.java:1731)
    at java.io.ObjectInputStream.readObject0(ObjectInputStream.java:1328)
    at java.io.ObjectInputStream.readObject(ObjectInputStream.java:350)
    ... 15 more


```

Original issue reported on code.google.com by `ireney.b...@gmail.com` on 18 Nov 2010 at 5:49

Attachments:
- [JsTestDriverFail.png](https://storage.googleapis.com/google-code-attachments/js-test-driver/issue-165/comment-0/JsTestDriverFail.png)

**Comment by GoogleCodeExporter** [comment]
Original comment by `aeagle22206` on 11 Apr 2011 at 7:32
- Added labels: **IntelliJ**

**Comment by GoogleCodeExporter** [comment]
```
If this bug is still reproducible in the plugin v1.3 or higher, could you 
please provide tests source (*.js and config file)?
```

Original comment by `Sergey.S...@gmail.com` on 10 May 2011 at 8:21

**Comment by GoogleCodeExporter** [comment]
```
It seems this bug is fixed in the plugin v1.3 or higher.
```

Original comment by `Sergey.S...@gmail.com` on 10 Jun 2011 at 10:20
- Changed state: **Fixed**
